src/core/fillOrder.ts

```diff
@@ -1,19 +1,25 @@
+import { Address } from 'viem'
 import { BundleEvent, Transaction } from '../types'
+import { debugLog } from '../helpers/logger'
+import { getOptimalDestinationChain } from './inventory'
 
 export const isClaimFirst = async (bundle: BundleEvent) => {
-  // todo
+  // todo: This could be enhanced with inventory-based decision making
```

Comment:
this is more to do with risk taking of whether the resource lock is used and potentially order amount

Reply:
oh! ok thanks, that helps me understand the purpose of this better